internal/datatransfer/monitor.go

```diff
@@ -228,6 +228,13 @@ func (m *Monitor) storeLastUpdateTime(workloadName string) {
 	m.lastSuccessfulSyncTimes[workloadName] = time.Now()
 }
 
+func (m *Monitor) ForceUpdate() error {
```

Comment:
WDYT about making it part of `Start`? After all it's supposed when the component starts.